src/components/BackToTop.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <v-btn
+    v-show="model"
```

Comment:
Does using `v-if` instead of `v-show` remove the need to have the pointer event managing classes? We shouldn't need to disable pointer events if the element isn't rendered in the DOM at all